### PR TITLE
Add Makefile and Debian packaging

### PR DESCRIPTION
**Pull request by NotKit**
I'm not very familiar with Debian packaging, so I used osso-bookmark-engine as base. Tested under Devuan.

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+libplayback (0.5-21+0m5) unstable; urgency=low
```
Comment (pali) [inline debian/changelog]: Last version in Maemo is 0_0.5-23+0m5. So you should have version higher then this.

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Architecture: any
+Section: libs
+Priority: optional
+Depends: ${shlibs:Depends}
+Description: OSSO Playback client library.
+
+Package: libplayback-1-dev
+Architecture: any
+Section: devel
+Priority: optional
```
Comment (pali) [inline debian/control]: Section and Priority can be removed as they are same as in Source part.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+LDFLAGS += `pkg-config --libs-only-L $(PKGDEPS)`
+LDLIBS += `pkg-config --libs-only-l --libs-only-other $(PKGDEPS)`
+
+LIBS=libplayback.la
+
+%.lo: src/%.c
+	libtool --tag=CC --mode=compile $(CC) $(CFLAGS) $(CPPFLAGS) -c $<
+
+libplayback.la: bluetooth.lo mute.lo playback.lo playback-types.lo privacy.lo
+	libtool --mode=link --tag=CC $(CC) $(LDFLAGS) -rpath $(libdir) -version-number 0:0:5 -o $@ $^ $(LDLIBS)
```
Comment (pali) [inline Makefile]: In Maemo library name is /usr/lib/libplayback-1.so.0.0.5

File: libplayback.pc [hunk 4]
```diff
@@ -0,0 +1,10 @@
+prefix=/usr
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+Name: libplayback
+Description: OSSO Playback client library
+Version: 0.0.5
+Libs: -L${libdir} -lplayback
```
Comment (pali) [inline libplayback.pc]: -lplayback-1

**Review by pali — CHANGES_REQUESTED** [review]
Hi! There are a few comments...

**Comment by freemangordon** [comment]
Hi and thanks for the pull request. However, it seems me and Pali got used to linux kernel ML review process and we are a bit picky :), so please, fix what Pali has requested and make a new pull request if not a problem.

Also, if you didn't already do it - you can reuse some parts (like .pc file and entries in debian/control etc.) from the binary packages in maemo - http://repository.maemo.org/pool/fremantle/nokia-binaries/4bc37c7c77ebe90177c050b805a8dc79/nokia-binaries/libp/libplayback/

File: debian/control (hunk 2 above)
Comment (NotKit) [reply debian/control]: Should Section be removed for -dev and -dbg packages as well? Source part has "Section: libs". I checked Maemo binary package, it has "Section: libdevel" for -dev.

Comment (pali) [reply debian/control]: If Section or Priority is not specified in Package part, then it is automatically taken from the Source part. So it does not make sense to re-define Section or Priority in Package part to same value.

I would say, that we should match what Maemo binaries have (if values are not bogus).

File: debian/control [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Build-Depends: debhelper (>= 5), libtool, pkg-config, libdbus-1-dev
+
+Package: libplayback-1-0
+Architecture: any
+Depends: ${shlibs:Depends}
+Description: OSSO Playback client library.
+
+Package: libplayback-1-dev
+Architecture: any
+Depends: libplayback-1-0
```
Comment (pali) [inline debian/control]: (= ${binary:Version})

File: debian/control [hunk 6]
```diff
@@ -0,0 +1,22 @@
+Source: libplayback
+Section: libs
+Priority: optional
+Maintainer: Ivaylo Dimitrov <ivo.g.dimitrov.75@gmail.com>
```
Comment (pali) [inline debian/control]: If you are going to maintain this Debian packaging (we would be happy if yes :-)) then fill free to put your name there.

**Review by pali — APPROVED** [review]

**Pull request merged**